x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/text_expansion_callout.tsx

```diff
@@ -123,14 +102,131 @@ export const TextExpansionCallOut: React.FC<TextExpansionCallOutProps> = (props)
               tagName="p"
             />
           </EuiText>
-          <EuiLink target="_blank" href={docLinks.elser}>
-            <FormattedMessage
-              id="xpack.enterpriseSearch.content.index.pipelines.textExpansionCallOut.learnMoreLink"
-              defaultMessage="Learn more"
-            />
-          </EuiLink>
+          <EuiFlexGroup direction="row" gutterSize="s" alignItems="center">
+            <EuiButton
```

Comment:
Elements under `EuiFlexGroup` should be `EuiFlexItem` to make them work properly. It may work as current way but normaly they should be wrapped.

Reply:
Done, I think now every element is wrapped into an `EuiFlexItem`.